src/shared/src/popup.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-underscore-dangle */
+/// <reference path="../../../typings/fs-base-ui/html_ui/JS/common.d.ts" />
```

Comment:
I think this is not needed anymore after the typings changes in #5359.

Reply:
![image](https://user-images.githubusercontent.com/15316958/129508868-78ef219b-9e90-475b-b739-85c9ab2c2139.png)

Still needed